data/CathodeParameters_KCsCb.txt

```diff
@@ -0,0 +1,36 @@
+# First line is: photocathode_model_choice cathode_thickness number_of_cathode_points
+# photocathode_model_choice: 0 = default model, 1 or 2 = new models
+# "1" is copied from geant4.11, which is based on https://ieeexplore.ieee.org/document/9875513
+# Model the alloy as a thin layer with real refractive index, then calculate reflection and transmission probability
+# Cannot handle total internal reflection when n1<n2
+# "2" is implemented based on https://arxiv.org/abs/physics/0408075v1
+# Model the alloy as a thin layer with real and imaginary refractive indices, then calculate absorption, reflection and transmission probability
+# cathode_thickness: unit = nm
+# number_of_cathode_points: number of points to read refractive indices (limited to <=100)
```

Comment:
```suggestion
# number_of_cathode_points: number of points of photon energy (rows below) to read refractive indices (limited to <=100)
```
Think it's important to say what a "point" is (e.g. it's photon energy, not position / something else). There might be better wording than this suggestion!

Reply:
Please add this to the two other txt files too

Reply:
Alternatively you could
- strip the file format description into another place (to avoid repetition / docs getting mismatched between files) into another place e.g. `doc/CathodeParameterFormat.md`
- in these `.txt` files, have a line like `# For a description of the format of this file, see $WCSIMDIR/doc/CathodeParameterFormat.md`